fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/SumoAmbassador.java

```diff
@@ -280,7 +280,7 @@ private void addNotYetAddedVehicles(long time) throws InternalFederateException
                 }
                 // TODO: Find better solution. Currently, an arbitrary SUMO route for external vehicles is selected, since a registered
                 //       SUMO route is required when adding a vehicle to SUMO. Using an empty route id "" leads to an error.
```

Comment:
This comment is wrong now (and was wrong before too?). I would write why we choose "" as the default route id.